app/src/main/java/com/kickstarter/viewmodels/LoginViewModel.kt

```diff
@@ -272,6 +273,6 @@ interface LoginViewModel {
 
         override fun tfaChallenge() = this.tfaChallenge
 
-        override fun showInterstitialFragment(): BehaviorSubject<AccessTokenEnvelope> = this.showInterstitial
```

Comment:
Do we need the access token on the interstitial screen for resending the email verification email?

Reply:
It has been removed, we do not need the accessToken on the interstitial. The resend email mutation works with the backend session, so we don't need any data to pass any data into the interstitial, that point was unclear when that screen was builded so doing a bit of refactor in here :)